addons/base_import_module/models/ir_module.py

```diff
@@ -218,6 +218,14 @@ def _import_module(self, module, path, force=False, with_demo=False):
                 _logger.info('module %s: no translation for language %s', module, lang)
         translation_importer.save(overwrite=True)
 
+        if ('knowledge.article' in self.env
+            and (article_record := self.env.ref(f"{module}.welcome_article", raise_if_not_found=False))
```

Comment:
Can't we remove the () ?

```suggestion
            and article_record := self.env.ref(f"{module}.welcome_article", raise_if_not_found=False)
```

Reply:
@ryv-odoo Could you help @pimodoo here?

Reply:
Don't understand why we want to remove the `()` here
`:=` is executed after `and`, if you just remove it, `article_record` will be used before assigned and the logic becomes different